guides/InstallationAdminGuide.md

```diff
@@ -523,14 +531,32 @@ Where:
 
 An example directive is:
 
-![alt text](ExampleDirective.png "Example Directive")
+```apache
+<VirtualHost *:80>
+  ServerName www.equella.com
+  ProxyPass / http://equellahost:80/ nocanon
```

Comment:
Might as well change these to the generic form as well

```apache
ServerName {external-server-name}
ProxyPass / http://{equella-host}:{http-port}/ nocanon
```

Reply:
Duplicate the example on line 518?

Reply:
Updated :ballot_box_with_check: